README_pt-br.md

```diff
@@ -0,0 +1,446 @@
+![](Assets/header2.jpg)
+
+<p align="center">
+    <a href="https://travis-ci.org/Juanpe/SkeletonView">
+      <img src="https://img.shields.io/travis/Juanpe/SkeletonView.svg">
+    </a>
+    <a href="https://instagram.github.io/IGListKit/">
+        <img src="https://img.shields.io/cocoapods/p/SkeletonView.svg" alt="Platforms">
+    </a>
+    <img src="https://img.shields.io/badge/Swift-4.1-orange.svg" />
+    <a href="https://cocoapods.org/pods/SkeletonView">
+        <img src="https://img.shields.io/cocoapods/v/SkeletonView.svg" alt="CocoaPods" />
+    </a>
+    <a href="https://github.com/Carthage/Carthage">
+        <img src="https://img.shields.io/badge/carthage-compatible-4BC51D.svg?style=flat" alt="Carthage" />
+    </a>
+    <a href="https://cocoapods.org/pods/SkeletonView">
+        <img src="https://img.shields.io/cocoapods/dt/SkeletonView.svg?style=flat" alt="CocoaPods downloads" />
+    </a>
+    <a href="https://twitter.com/JuanpeCatalan">
+        <img src="https://img.shields.io/badge/contact-@JuanpeCatalan-blue.svg?style=flat" alt="Twitter: @JuanpeCatalan" />
+    </a>
+    <a href="https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=MJ4Y2D9DEX6FL&lc=ES&item_name=SkeletonView&currency_code=EUR&bn=PP%2dDonationsBF%3abtn_donate_SM%2egif%3aNonHosted">
+        <img src="https://img.shields.io/badge/Donate-PayPal-green.svg" alt="Paypal" />
+    </a>
+    <a href="https://twitter.com/intent/tweet?text=Wow%20This%20library%20is%20awesome:&url=https%3A%2F%2Fgithub.com%2FJuanpe%2FSkeletonView">
+      <img src="https://img.shields.io/twitter/url/https/github.com/Juanpe/SkeletonView.svg?style=social" alt="License" />
+    </a>
+    <a href="https://twitter.com/JuanpeCatalan">
+      <img src="https://img.shields.io/twitter/follow/JuanpeCatalan.svg?style=social&label=Follow" alt="Twitter" />
+    </a>
+</p>
+
+🌎  Traduções: </br>
+[🇨🇳](https://github.com/Juanpe/SkeletonView/blob/master/README_zh.md) por [@WhatsXie](https://twitter.com/WhatsXie) </br>
+[🇧🇷](https://github.com/Juanpe/SkeletonView/blob/master/README_zh.md) por [@brunomunizaf](https://twitter.com/brunomuniz_af)
```

Comment:
Here we should specify the original link. You can check it on Chinese README file

Reply:
On the original: both translations are linked. On each translated readme file I inserted the links for the original and the other translation. LGTY?